katex-header.html

```diff
@@ -0,0 +1,8 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css" integrity="sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3+Aro6EYUG4+cU+KJWu/X" crossorigin="anonymous">
+<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js" integrity="sha384-g7c+Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI+sEnkvrMWph2EDg4" crossorigin="anonymous"></script>
+<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js" integrity="sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC+Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>
```

Comment:
Script handler attributes are a security smell, it would be better to have hashed/nonce'd script to bind the event on the script element, ala

```
<script id="autorender" src="...">

<script nonce="${nonce}">
document.addEventListener('DOMContentLoaded', function () {
  document.getElementById('autorender')
          .addEventListener('load', function doThings() { renderMathInElement(document.body)});
});
</script>
```

Reply:
Sure, I just followed [these docs](https://katex.org/docs/autorender.html), feel free to make whatever changes you want to the configuration.

Reply:
@dconnolly WDYT of merging this as-is so that we can render math immediately and improving the script configuration later? It would be nice to do best practices but I feel that the risk of XSS or similar on our static docs is pretty low.

Reply:
Yeah that's fine with me 👍